src/internal/domain/CSSRuleWrapper.ts

```diff
@@ -81,7 +84,13 @@ export const prefixWrapCSSSelector = (
 
     // Anything other than a root tag is always prefixed.
     if (isNotRootTag(cleanedSelector)) {
-        return prefixSelector + " " + cleanedSelector;
+        let result = prefixSelector + " " + cleanedSelector;
+
+        if (appendCompoundSelector) {
+            result += `, ${prefixSelector}${cleanedSelector}`;
+        }
+
+        return result;
     }
 
     // Handle special case where root tags should be converted into classes
```

Comment:
Looks great! I believe the root tags should also be appended when prefixRootTags is enabled.
```ts
if (prefixRootTags) {
    let result = prefixSelector + " ." + cleanedSelector;
    
    if (appendCompoundSelector) {
          result += `, ${prefixSelector}.${cleanedSelector}`;
    }

    return result;
}
```

Reply:
Maybe like this
```ts
let cleanedSelector = cleanSelector(cssSelector);

const isNotRootTag = isNotRootTag(cleanedSelector);
if (!isNotRootTag && prefixRootTags) {
    cleanedSelector = "." + cleanedSelector;
}

if (isNotRootTag || prefixRootTags) {
    let result = prefixSelector + " " + cleanedSelector;

    if (appendCompoundSelector) {
        result += `, ${prefixSelector}${cleanedSelector}`;
    }

    return result;
}
```

Reply:
Hi @oskar-anderson, I have applied an update which takes into the `prefixRootTags` option. Would you mind please re-reviewing.

Reply:
Maybe rename `appendCompoundSelectorToSelector` to `generateSelector` as it also handles the regular append case or in case you like oneliners replace the function with this:
```
[`${prefixSelector} ${cleanedSelector}`, appendPrefixSelector ? `${prefixSelector}${cleanedSelector}` : ""].filter(x => x !== "").join(", ")
```

Reply:
I realized that a compound selector will only work for class selectors and not html tags. When one selector is invalid the entire rule becomes invalid. This becomes a problem with libraries, Ex. flatpickr uses `span` selectors for styling. I think this feature might have been a waste of time 😞.

Reply:
All good, I will put this MR on ice for now.